doc/nrf/releases_and_maturity/releases/release-notes-2.9.0.rst

```diff
@@ -0,0 +1,661 @@
+
+* `Twister test report for nRF Connect SDK v2.9.0`_
+* `Hardware test report for nRF Connect SDK v2.9.0`_
+
+IDE and tool support
+********************
+
+`nRF Connect extension for Visual Studio Code <nRF Connect for Visual Studio Code_>`_ is the recommended IDE for |NCS| v2.9.0.
+See the :ref:`installation` section for more information about supported operating systems and toolchain.
+
+Supported modem firmware
+************************
+
+See the following documentation for an overview of which modem firmware versions have been tested with this version of the |NCS|:
+
+* `Modem firmware compatibility matrix for the nRF9151 DK`_
+* `Modem firmware compatibility matrix for the nRF9161 DK`_
+* `Modem firmware compatibility matrix for the nRF9160 DK`_
+
+Use the latest version of the `Programmer app`_ of `nRF Connect for Desktop`_ to update the modem firmware.
+See :ref:`nrf9160_gs_updating_fw_modem` for instructions.
+
+Modem-related libraries and versions
+====================================
+
+.. list-table:: Modem-related libraries and versions
+   :widths: 15 10
+   :header-rows: 1
+
+   * - Library name
+     - Version information
+   * - Modem library
+     - `Changelog <Modem library changelog for v2.9.0_>`_
+   * - LwM2M carrier library
+     - `Changelog <LwM2M carrier library changelog for v2.9.0_>`_
+
+
+Known issues
+************
+
+Known issues are only tracked for the latest official release.
+See `known issues for nRF Connect SDK v2.9.0`_ for the list of issues valid for the latest release.
+
+Migration notes
+***************
+
+See the `Migration guide for nRF Connect SDK v2.9.0`_ for the changes required or recommended when migrating your application from |NCS| v2.8.0 to |NCS| v2.9.0.
+
+.. _ncs_release_notes_290_changelog:
+
+Changelog
+*********
+
+The following sections provide detailed lists of changes by component.
+
+IDE, OS, and tool support
+=========================
+
+* Added support for the Nordic Thingy:91 X to the `Quick Start app`_.
+  The list on the :ref:`gsg_guides` page is updated accordingly.
+
+* Updated the deprecation notes for `nRF Command Line Tools`_ added in the previous release.
+  The notes now clearly state the tools will be archived, no updates will be made to the software, but it will still be available for download.
+
+Board support
+=============
+
+* Updated various tests and samples to use Zephyr's :ref:`native simulator <zephyr:native_sim>` instead of Zephyr's :ref:`native POSIX <zephyr:native_posix>` for :ref:`running_unit_tests`.
+  This mirrors the deprecation of ``native_posix`` in Zephyr.
+  Support for ``native_posix`` will be removed in Zephyr with the v4.2 release.
+  In the |NCS|, it will be removed once Zephyr v4.2 is upmerged to sdk-nrf.
+
+Build and configuration system
+==============================
+
+* Fixed the issue in the ``nordic-bt-rpc`` snippet, where an invalid memory map was created for nRF54H20 devices, which resulted in a runtime failure.
```

Comment:
Hi @peknis, please move this entry back to the changelog, it is relevant for 2.9-nRF54H20.